py/server/tests/test_multijoin.py

```diff
@@ -0,0 +1,81 @@
+        self.static_tableB = self.static_tableA.update(["c2=c1+1","d2=d1+2","e2=e1+3"]).drop_columns(["c1","d1","e1"])
+        self.test_update_graph = get_exec_ctx().update_graph
+        with update_graph.exclusive_lock(self.test_update_graph):
+            self.ticking_tableA = time_table("PT00:00:00.001").update(
+                ["a = i", "b = i*i % 13", "c1 = i * 13 % 23", "d1 = a + b", "e1 = a - b"]).drop_columns(["Timestamp"])
+            self.ticking_tableB = self.ticking_tableA.update(["c2=c1+1","d2=d1+2","e2=e1+3"]).drop_columns(["c1","d1","e1"])
+
+    def tearDown(self) -> None:
+        self.static_tableA = None
+        self.static_tableB = None
+        self.ticking_tableA = None
+        self.ticking_tableB = None
+        super().tearDown()
+
+    def test_static_simple(self):
+        mj_table = multi_join(input=[self.static_tableA, self.static_tableB], on=["a","b"])
+
+        # Output table is static
+        self.assertFalse(mj_table.table().is_refreshing)
+        # Output table has same # rows as sources
+        self.assertEqual(mj_table.table().size, self.static_tableA.size)
+        self.assertEqual(mj_table.table().size, self.static_tableB.size)
+
+
+    def test_ticking_simple(self):
+        mj_table = multi_join(input=[self.ticking_tableA, self.ticking_tableB], on=["a","b"])
+
+        # Output table is refreshing
+        self.assertTrue(mj_table.table().is_refreshing)
+        # Output table has same # rows as sources
+        with update_graph.exclusive_lock(self.test_update_graph):
+            self.assertEqual(mj_table.table().size, self.ticking_tableA.size)
+            self.assertEqual(mj_table.table().size, self.ticking_tableB.size)
+
+
+    def test_static(self):
+        mj_input = [
+            MultiJoinInput(table=self.static_tableA, on=["key1=a","key2=b"], joins=["c1","e1"]),
+            MultiJoinInput(table=self.static_tableB, on=["key1=a","key2=b"], joins=["d2"])
+        ]
+        mj_table = multi_join(mj_input)
+
+        # Output table is static
+        self.assertFalse(mj_table.table().is_refreshing)
+        # Output table has same # rows as sources
+        self.assertEqual(mj_table.table().size, self.static_tableA.size)
+        self.assertEqual(mj_table.table().size, self.static_tableB.size)
+
+
+    def test_ticking(self):
+        mj_input = [
+            MultiJoinInput(table=self.ticking_tableA, on=["key1=a","key2=b"], joins=["c1","e1"]),
+            MultiJoinInput(table=self.ticking_tableB, on=["key1=a","key2=b"], joins=["d2"])
+        ]
+        mj_table = multi_join(mj_input)
+
+        # Output table is refreshing
+        self.assertTrue(mj_table.table().is_refreshing)
+        # Output table has same # rows as sources
+        with update_graph.exclusive_lock(self.test_update_graph):
+            self.assertEqual(mj_table.table().size, self.ticking_tableA.size)
+            self.assertEqual(mj_table.table().size, self.ticking_tableB.size)
+
+if __name__ == '__main__':
```

Comment:
I don't see any tests that take a single Table or a single MultiJoinInput.

Reply:
Added single table instantiation for all tests.